### ISSUE TITLE
Shouldn't the demo.html file have the example js line to include, pulling from the cdn?

### ISSUE THREAD

**Issue by katrialesser** [byuweb/byu-hero-banner#16]
I can figure it out, but we've told people what this line was in our other demos for ease and clarity.

**Comment by kebowdog** [comment]
That makes sense. Thus far I'd been relying on the README, but we could put it in the demo.html as well.

**Comment by ThatJoeMoore** [comment]
I don't think that our demos are necessarily the best place to put specifics on documentation. There are just a lot of problems with embedding CDN links instead of relative links in the demos, such as "what happens when the version on 'latest' behaves differently than the version of the demo I'm viewing?"  The README is typically the canonical place for documentation; the demo is just a demo.  That said, we should work on a long-term approach for more formal documentation of our components and libraries, possibly using tools like [JSDoc](https://www.npmjs.com/package/jsdoc), [iron-component-page](https://www.webcomponents.org/element/PolymerElements/iron-component-page), and [iron-doc-*](https://github.com/PolymerElements/iron-doc-viewer).
